template/composer.json

```diff
@@ -28,6 +27,7 @@
     "behat/mink-goutte-driver":     "*",
     "behat/mink-selenium2-driver":  "*",
     "behat/mink-browserkit-driver": "*",
+    "drush/drush":                  "dev-master",
```

Comment:
Ah, I hadn't considered that. Will running `drush @alias status` use the drush bin in the remote application's vendor dir?